docs/book/src/clusterctl/provider-contract.md

```diff
@@ -306,15 +306,16 @@ functioning of all the `clusterctl` features both in single tenancy or multi-ten
 
 Provider authors should be aware that `clusterctl move` command implements a discovery mechanism that considers:
 
-* All the objects of Kind defined in one of the CRDs installed by clusterctl using `clusterctl init`. 
+* All the objects of Kind defined in one of the CRDs installed by clusterctl using `clusterctl init`.
 * `Secret` and `ConfigMap` objects.
-* the `OwnerReference` chain of the above objects.
+* The `OwnerReference` chain of the above objects.
+* Any object with the with the "force move" label (`clusterctl.cluster.x-k8s.io/force-move`)attached to it.
```

Comment:
nit:
```suggestion
* Any object with the with the "force move" label (`clusterctl.cluster.x-k8s.io/force-move`) attached to it.
```

Reply:
What happens if the object that has the move label on it is a parent to another object? Do we ignore the children of the moved object?

Reply:
Great question and I'm not sure I have the best answer to it. I've been assuming that any and all resource I want to move will be tagged with the force-move label. Seemed fair to me that devs wanting to use this would have to handle that tagging manually.

Reply:
That makes sense however maybe we can document it just in case users don't expect the ownership behavior to extend to resources marked with this label as well.

Reply:
Added a note to the doc in amended commit.